code/modules/projectiles/guns/energy/dueling.dm

```diff
@@ -159,6 +167,9 @@
 
```

Comment:
Make it so attacking another dueling pistol while !duel sets up a new dueling datum.

Reply:
It's still useless because there's never a second one

Reply:
>that way admins can spawn them in easier